<ui version="4.0" >
 <class>prefs_identity</class>
 <widget class="QWidget" name="prefs_identity" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>375</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>11</number>
   </property>
   <property name="topMargin" >
    <number>11</number>
   </property>
   <property name="rightMargin" >
    <number>11</number>
   </property>
   <property name="bottomMargin" >
    <number>11</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string comment="@label:chooser" >Default Language:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string comment="@label:textbox" >Email:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>kcfg_Email</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel2_2" >
     <property name="text" >
      <string comment="@label:textbox" >Name:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="kcfg_authorName" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string comment="@label:textbox" >Default Mailing List:</string>
     </property>
     <property name="buddy" >
      <cstring>kcfg_DefaultMailingList</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="kcfg_DefaultMailingList" />
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="DefaultLangCode" />
   </item>
   <item row="7" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="kcfg_authorEmail" />
   </item>
   <item row="5" column="1" >
    <widget class="QLineEdit" name="kcfg_DefaultLangCode" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLineEdit" name="kcfg_authorLocalizedName" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string comment="@label:textbox" >Localized Name:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>kcfg_authorName</tabstop>
  <tabstop>kcfg_authorEmail</tabstop>
  <tabstop>DefaultLangCode</tabstop>
  <tabstop>kcfg_DefaultMailingList</tabstop>
  <tabstop>kcfg_authorLocalizedName</tabstop>
  <tabstop>kcfg_DefaultLangCode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
